lib/routes/actors.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable new-cap */
+const router = require('express').Router();
+const Actor = require('../models/actor');
+const Film = require('../models/film');
+
+router
+
+  .post('/', (req, res, next) => {
+  
+    Actor.create(req.body)
+      .then(actor => {
+        console.log(actor);
+      
+        res.json(actor);
+      })
+      .catch(next);
+  })
+
+  .get('/', (req, res, next) => {
```

Comment:
it's good that you have the get routes - looks like you are just missing tests for them